CalibCalorimetry/HcalAlgos/src/HcalTimeSlew.cc

```diff
@@ -8,22 +8,25 @@ static const double cap = 6.0;
 static const double tspar0[2] = {15.5, 12.2999};
 static const double tspar1[2] = {-3.2,-2.19142};
 static const double tspar2[2] = {32, 0};
-static const double tspar0_siPM[2] = {3., 3.}; // 3ns delay for MC and DATA, recheck later for data
+static const double tspar0_siPM[2] = {0., 0.}; // 2ns delay for MC and DATA, recheck later for data
```

Comment:
@fwyzard reching, the enum "source==data/MC/InputPars" is different in the various if statement